wrs/index.js

```diff
@@ -1 +1,69 @@
 import './style.css';
+
+let leaderboard = [];
+
+const user = document.getElementById('Inputname');
+const score = document.getElementById('Inputscore');
+const pointsDeploy = document.querySelector('.table_container');
+
+const getScores = async () => {
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/GARdimyGDm/scores');
+    const json = await response.json();
+    leaderboard = json.result;
+
+    // Sort the leaderboard based on scores in descending order
+    leaderboard.sort((a, b) => b.score - a.score);
+
+    pointsDeploy.innerHTML = '';
+    leaderboard.forEach((dat) => {
+      const row = document.createElement('tr');
+      const name = document.createElement('td');
+      name.className = 'point';
+      name.textContent = `${dat.user}: ${dat.score}`;
+      row.appendChild(name);
+      pointsDeploy.appendChild(row);
+    });
+  } catch (error) {
+    // Handle the error
+  }
+};
+
+const addScore = async (userName, userScore) => {
+  if (!userName || !userScore) {
+    // Display an error message or perform appropriate action for empty inputs
+    return;
+  }
+
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/GARdimyGDm/scores', {
+      method: 'POST',
+      body: JSON.stringify({
```

Comment:
Well done with your final touches. 💯 

[OPTIONAL] I noticed that each score is rendered twice. It would be best to look at it and make sure that your does not add double scores.
![image](https://github.com/Gardimy/Leaderboard_project/assets/38088818/5a755014-19e0-4f5f-b21c-0dbc09ff6a20)